pages/common/gitlab-runner.md

```diff
@@ -0,0 +1,28 @@
+# gitlab-runner
+
+> Run GitLab CI jobs.
+> More information: <https://docs.gitlab.com/runner/>.
+
+- Register a runner:
+
+`sudo gitlab-runner register --url {{https://gitlab.example.com/}} --registration-token {{token}} --name {{runner_name}}`
```

Comment:
Based on https://docs.gitlab.com/runner/register/index.html the trailing `/` is not required.
Also if I wonder if `runner_name` is not redundant. Could we use just `{{name}}`? or just `runner_name` for the first example:
```suggestion
`sudo gitlab-runner register --url {{https://gitlab.example.com}} --registration-token {{token}} --name {{name}}`
```